src/sentry/integrations/slack/unfurl/discover.py

```diff
@@ -220,9 +220,10 @@ def unfurl_discover(
             chart_url=url,
         ).build()
 
+    org_model = integration.organizations.first()
     analytics.record(
         "integrations.slack.chart_unfurl",
-        organization_id=integration.organizations.all()[0].id,
+        organization_id=org_model.id if org_model is not None else None,
```

Comment:
`organization_id` is implicitly required.
```py
class IntegrationSlackChartUnfurl(analytics.Event):
    type = "integrations.slack.chart_unfurl"

    attributes = (
        analytics.Attribute("user_id", required=False),
        analytics.Attribute("organization_id"),
        analytics.Attribute("unfurls_count", type=int),
    )
```

Reply:
Would it make sense to not record the analytic if org_id isn't visible?